apps/prairielearn/python/prairielearn/core.py

```diff
@@ -917,7 +907,7 @@ def numpy_to_matlab(
 
 
 def string_from_numpy(
-    A: npt.NDArray[Any] | _NumPyScalarType,
+    A: numbers.Number | _NumPyScalarType | npt.NDArray[Any],
```

Comment:
I added `numbers.Number` for consistency with `latex_from_2darray`, which does actually need that as a valid argument type.

Reply:
As of 1d83004, this now uses the new `_NumericScalarType`.